.github/actions/send-telegram-notify/action.yml

```diff
@@ -42,6 +42,45 @@ runs:
         curl -s https://api.github.com/repos/tarantool/tarantool/actions/runs/${{ github.run_id }}/jobs | jq -r '.jobs[0].id' | tee -a $GITHUB_ENV
         echo 'EOF' >> $GITHUB_ENV
       shell: bash
+    # convert commit message to single line with marked new lines as 'TELEGRAMNEWLINE'
+    - env:
+        COMMIT_MSG: |
+          ${{ github.event.head_commit.message }}
+      run: |
+        # Make changes in the commit message to avoid of its fail on Markdown:
+        # 1. Convert message from multi lines to single line.
+        # 2. Add backslashes to single quote marks in message.
+        # 3. Add backslashes to '`' in message.
+        if ${{ env.OS_IS_FEDORA_OR_RHEL == 'true' }} || ${{ env.KERNEL_NAME == 'Darwin' }} ; then
+          msg="${COMMIT_MSG//$'\n'/TELEGRAMNEWLINE}"
+          msg="${msg//$'\\\\'/BACKSLASH}"
+          msg="${msg//$'\\\''/\'}"
+          msg="${msg//$'`'/BACKTICK}"
+        else
+          msg="${COMMIT_MSG//$'\n'/TELEGRAMNEWLINE}"
+          msg="${msg//$'\\'/BACKSLASH}"
+          msg="${msg//$'\''/\\\'}"
+          msg="${msg//$'`'/BACKTICK}"
+        fi
```

Comment:
Why are we changing "\" to "BACKSLASH" and "`" to "BACKTICK"? 
I suggest replacing them with "/" and "'" for a more readable telegram message.

Reply:
I don't think that change with meaning of the commit message, like:
```
'`' -> '''
```
is better than not so beautiful, but accurate change, like:
```
'`' -> 'BACKTICK'
```

Also I've used the commit message of the 3rd commit ('github-ci: fix message send on rhel/fedora hosts') to check sending message. I saw that the message was changed in readable condition, like this part:
```
'send-telegram-notify' action creates message with syntax error:

 --------------'BACKSLASHn't'BACKSLASHn'BACKTICKBACKTICKBACKTICK'BACKSLASHn'')) ; BACKSLASH
 ^
 SyntaxError: unexpected character after line continuation character

It happened because of extra quotes at 'BACKSLASHn' while it had to be BACKSLASHn.
To avoid of it the same message changes must be done as for OSX
```